services.md

```diff
@@ -109,6 +109,16 @@ The following services are available on the Yggdrasil network, courtesy of our c
 - Other channels can be listed with description as their topic
 - Also accessible via cjdns at `fc42:7cfa:b830:e988:f192:717f:6576:ed12` port `6667` (TCP)
 
+### BonoboNET
+
+- Main channel is `#general`
+- Accessible via the public internet at `41.157.98.109` port `9006`
+- On yggdrasil at `200:c2a4:4104:ce7b:c778:46ef:e914:96bf`
```

Comment:
Might be good to clarify the port numbers for the Yggdrasil connections too :-)

Reply:
Will do

Reply:
There ya go @neilalexander